service/service.go

```diff
@@ -160,7 +166,7 @@ func New(config Config) (*Service, error) {
 			TenantCluster: tenantCluster,
 
 			CRDLabelSelector: config.Viper.GetString(config.Flag.Service.CRD.LabelSelector),
-			ProjectName:      config.Name,
+			ProjectName:      project.Name(),
```

Comment:
Rather than having each project package on each controller file, I think this is cleaner.